### PR TITLE
Auxilio Emergencial command

### PR DESCRIPTION
**Pull request by Doges**
yeah, i did it

brazil

### CODE REVIEW

File: src/utils/Constants.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ module.exports = {
   CHOCOLATEY_COLOR: '#7eb5e2',
   SNAPCRAFT_COLOR: '82BEA0',
   JETBRAINS_PLUGIN: '#d44386',
-  ECLIPSE_PLUGINS_COLOR: '#42347f',
```
Comment (davipatury) [inline src/utils/Constants.js:33]: Era pra tirar essa linha mesmo?

**Review by davipatury — DISMISSED** [review]

File: src/utils/Constants.js (hunk 1 above)
Comment (almeidx) [reply src/utils/Constants.js:33]: nope

Comment (almeidx) [reply src/utils/Constants.js:33]: https://github.com/SwitchbladeBot/switchblade/blob/85cc04f6042b603fb96ccb5081d5b408a1f159b2/src/commands/misc/eclipseplugin.js#L10

**Review by davipatury — APPROVED** [review]
muito bom nota 0

**Comment by Doges** [comment]
> 
> 
> muito bom nota 0

**Review by moruzerinho6 — APPROVED** [review]
image

Do not approve doges PR, it's the worst thing I've ever done!

File: src/commands/government/auxilioemergencial.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  async run ({ t, author, channel, language }, cpf) {
+    const auxilio = await this.client.apis.portaldatransparencia.searchAuxilioCPF(cpf)
+    moment.locale(language)
+    try {
+      channel.send(
+        new SwitchbladeEmbed(author)
+          .setAuthor('Portal da Transparência', 'https://logodownload.org/wp-content/uploads/2017/03/brasao-do-brasil-republica-1.png', 'http://portaltransparencia.gov.br/')
+          .setColor(Constants.PORTALTRANSPARENCIA_COLOR)
+          .setDescription(`**${t('commands:auxilioemergencial.name')}**: ${auxilio[0].beneficiario.nome}\n**${t('commands:auxilioemergencial.cpf')}**:` + auxilio[0].beneficiario.cpfFormatado + `\n**${t('commands:auxilioemergencial.nis')}**: ${auxilio[0].beneficiario.nis}\n**${t('commands:auxilioemergencial.amount')}**: R$ ${auxilio[0].valor}.00\n**${t('commands:auxilioemergencial.city')}**: ${auxilio[0].municipio.nomeIBGE} - ${auxilio[0].municipio.uf.sigla}`)
```
Comment (metehus) [inline src/commands/government/auxilioemergencial.js:29]: clear pls
```suggestion
          .setDescription(`**${t('commands:auxilioemergencial.name')}**: ${auxilio[0].beneficiario.nome}
          **${t('commands:auxilioemergencial.cpf')}**:` + auxilio[0].beneficiario.cpfFormatado
          + `\n**${t('commands:auxilioemergencial.nis')}**: ${auxilio[0].beneficiario.nis}
          **${t('commands:auxilioemergencial.amount')}**: R$ ${auxilio[0].valor}.00
          **${t('commands:auxilioemergencial.city')}**: ${auxilio[0].municipio.nomeIBGE} - ${auxilio[0].municipio.uf.sigla}`)
```

**Review by metehus — CHANGES_REQUESTED** [review]

File: src/commands/government/auxilioemergencial.js (hunk 2 above)
Comment (metehus) [reply src/commands/government/auxilioemergencial.js:29]: fodase

Comment (almeidx) [reply src/commands/government/auxilioemergencial.js:29]: too late

**Pull request merged**